### PR TITLE
KEP-4292: Custom profile in kubectl debug

### PR DESCRIPTION
**Pull request by ardaguclu**
This PR adds new environment variable into the docs for https://github.com/kubernetes/enhancements/issues/4292

### CODE REVIEW

**Comment by Vyom-Yadav** [comment]
Hey @ardaguclu 👋 please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday March 12th 2024 18:00 PST. Thank you!

**Comment by ardaguclu** [comment]
@eddiezane would you mind tagging this from cli POV?. Thanks.

File: content/en/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -377,6 +377,14 @@ kubectl [flags]
 </td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_DEBUG_CUSTOM_PROFILE</td>
```
Comment (sftim) [inline content/en/docs/reference/kubectl/kubectl.md:381]: Thanks, but: I think we prefer these variables in alphabetical order. Could you make them sorted? (I can see they are already close).

Comment (ardaguclu) [reply content/en/docs/reference/kubectl/kubectl.md]: You are right, moved it to correct place in alphabetical order.

File: content/en/docs/reference/kubectl/kubectl.md [hunk 2]
```diff
@@ -350,6 +350,14 @@ kubectl [flags]
 <td></td><td style="line-height: 130%; word-wrap: break-word;">When set to false, turns off extra HTTP headers detailing invoked kubectl command (Kubernetes version v1.22 or later)</td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_DEBUG_CUSTOM_PROFILE</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, custom flag will be enabled in kubectl debug. This flag is used to customize the pre-defined profiles.
```
Comment (sftim) [inline content/en/docs/reference/kubectl/kubectl.md:357]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, custom flag will be enabled in <tt>kubectl debug</tt>. This flag is used to customize the pre-defined profiles.
```

**Review by sftim — COMMENTED** [review]
LGTM for docs

Tech review in https://github.com/kubernetes/website/pull/45223#issuecomment-1983482876

/lgtm
/approve

**Pull request merged**